tests/tests_app/conftest.py

```diff
@@ -20,6 +20,8 @@
     "template_react_ui": "https://github.com/Lightning-AI/lightning-template-react.git",
 }
 
+os.environ["LIGHTNING_DISPATCHED"] = "1"
+
```

Comment:
Why is it needed to set this here?

Reply:
It is safer in case you launch lightning with a debugger attached.